### PR TITLE
Dashboard of users in progress

### PR DESCRIPTION
**Pull request by anabergese**
Yanik needs my advance in the code to continue working in the dashboard of users (users profile)

### CODE REVIEW

File: app/views/celebrities/show.html.erb [hunk 1]
```diff
@@ -9,12 +9,6 @@
     <button class="button text-center text-white">
       <%= link_to 'Book now', celebrities_path, class: "no-deco" %>
     </button>
-    <button class="button text-center text-white">
```
Comment (Trinideachaval) [inline app/views/celebrities/show.html.erb]: keep the buttons

**Review by anabergese — COMMENTED** [review]
I keep the buttons

**Review by Trinideachaval — APPROVED** [review]
Perfect!

**Review by Trinideachaval — APPROVED** [review]
Perect!

**Pull request merged**